lib/Runtime/Language/AsmJsLink.cpp

```diff
@@ -143,18 +146,21 @@ namespace Js{
         }
 
 #ifdef ENABLE_SIMDJS
-        Var asmSimdObject = JavascriptOperators::OP_GetProperty(stdlib, PropertyIds::SIMD, scriptContext);
-        for (int i = 0; i < AsmJsSIMDBuiltinFunction::AsmJsSIMDBuiltin_COUNT; i++)
+        if (!simdBuiltinUsed.IsAllClear())
         {
```

Comment:
In Edge if the global object is used for stdlib, yes. Emscripten doesn't do this, so none of our benchmarks/large browser tests ran into this.